extras/rya.console/src/main/java/mvm/rya/shell/RyaAdminCommands.java

```diff
@@ -125,8 +126,10 @@ public boolean arePCJCommandsAvailable() {
         // that is new enough to use the RyaDetailsRepository and is configured to maintain PCJs.
         final ShellState shellState = state.getShellState();
         if(shellState.getConnectionState() == ConnectionState.CONNECTED_TO_INSTANCE) {
-            //TODO: if Mongo, return false.
             final GetInstanceDetails getInstanceDetails = shellState.getConnectedCommands().get().getGetInstanceDetails();
+            if(getInstanceDetails instanceof MongoGetInstanceDetails) {
```

Comment:
This is kind of a hack. Why not just store the type of repository we're connected to in the shell state object and key off of that?